cmake/modules/NSIS.template.in

```diff
@@ -766,6 +772,11 @@ Function .onInstSuccess
       WriteRegDWORD HKLM "Software\${APPLICATION_VENDOR}\${APPLICATION_NAME}" "skipUpdateCheck" "1"
    ${EndIf}
 
+   ; Viel hilft viel. The uninstaller should have already removed it..
```

Comment:
I'd happer if you said: "TODO: Only needed to when updating from 2.1.{0,1}. Remove in due time.